packages/dashboard-core-plugins/src/panels/NotebookPanel.jsx

```diff
@@ -665,6 +699,31 @@ class NotebookPanel extends Component {
     this.focus();
   }
 
+  /**
+   * Transform the link URI to load from where the notebook is if it's relative
+   * @param {String} src The link to transform
+   * @returns String the transformed link
+   */
+  handleTransformLinkUri(src) {
+    const { notebooksUrl } = this.props;
+    const { fileMetadata } = this.state;
+
+    if (src.charAt(0) === '/') {
+      return `${notebooksUrl}${src}`;
```

Comment:
This still duplicates the `/`...

Reply:
I'm an idiot. I didn't realize the trailing slash was being added at the bottom in the defaultProp path, I was looking in the .env (which doesn't have trailing slash).